argocon-demos/2025-automated-resilience-using-argo-events-for-real-time-incident-remediation/example/sensor-rbac.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: operate-workflow-sa
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: operate-workflow-role
+  namespace: argo
+rules:
+  - apiGroups:
+      - argoproj.io
+    verbs:
+      - "*"
```

Comment:
maybe a comment here or in the readme to remind people not to blindly run this in prod without understanding the permissions?